### PR TITLE
ECIP-1053 Add OpenRPC Service Discovery To JSON-RPC Services

### PR DESCRIPTION
**Pull request by soc1c**
To stay in sync with https://github.com/etclabscore/ECIPs and announce used ECIP numbers.

- [x] ECIP-1053 Add OpenRPC Service Discovery To JSON-RPC Services
- [ ] ~~ECIP-1054 Support for ETH Byzantium EVM and Protocol Upgrades~~
- [ ] ~~ECIP-1055 Succinct PoW Using Merkle Mountain Ranges~~
- [ ] ~~ECIP-1056: Support for ETH Constantinople EVM and Protocol Upgrades~~

Commit history might look weird as we also synced it with ethereumproject/ECIPs, I would suggest not to squash-merge this.

### CODE REVIEW

**Comment by soc1c** [comment]
Let's move on with this ECIP-1053 and sync the proposals one by one. :+1:

**Comment by phyro** [comment]
I'm all for this :+1:

File: ECIPs/ECIP-1053.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Add OpenRPC Service Discovery To JSON-RPC Services
+
+    ECIP: 1053
+    Title: Add OpenRPC Service Discovery To JSON-RPC Services
+    Author: Shane Jonas, Zachary Belford
+    Status: DRAFT
```
Comment (YazzyYaz) [inline ECIPs/ECIP-1053.md]: tiny nitpick that can be ignored., but can we make `DRAFT` to be `Draft`

Comment (soc1c) [reply ECIPs/ECIP-1053.md]:
```suggestion
    Status: Draft
```

**Comment by BelfordZ** [comment]
🎉

**Pull request merged**